broker/src/main/java/org/apache/rocketmq/broker/BrokerController.java

```diff
@@ -329,6 +331,10 @@ public boolean initialize() throws CloneNotSupportedException {
                 Executors.newFixedThreadPool(this.brokerConfig.getConsumerManageThreadPoolNums(), new ThreadFactoryImpl(
                     "ConsumerManageThread_"));
 
+            this.allocateMessageQueueExecutor =
+                Executors.newFixedThreadPool(this.brokerConfig.getConsumerManageThreadPoolNums(), new ThreadFactoryImpl(
+                    "AllocateMessageQueueThread_"));
```

Comment:
IMO, we don’t need to open a thread pool separately, could we reuse consumerManageExecutor?

Reply:
Updated. The processor thread had been deleted and the code had migrated to ConsumerManageProcessor.